scoring.js

```diff
@@ -36,8 +36,30 @@ function calculateReturnScore(player) {
 }
 
 module.exports.calculateScore = function (player) {
-  return calculatePassingScore(player) +
-    calculateRushingScore(player) +
-    calculateReceivingScore(player) +
-    calculateReturnScore(player)
-}
+  // if (player.position === 'QB') {
```

Comment:
It's good to keep code around like this when working on it but in practice always delete the unneeded code before you commit. Commented out code is just clutter once we've got things working.